app/vue/contexts/profile/ProfileDetailsPageContext.js

```diff
@@ -249,6 +271,56 @@ export default class ProfileDetailsContext extends BaseFuroContext {
     }
   }
 
+  /**
+   * Fetch profile orders.
+   *
+   * @param {{
+   *   address: string | null
+   * }} params - Parameters
+   * @returns {Promise<void>}
+   */
+  async fetchProfileOrders ({
+    address,
+  }) {
+    if (!address) {
+      return
+    }
+
+    this.statusReactive.isLoadingProfileOrders = true
+
+    const headers = {
+      accept: 'application/json',
+    }
+    const searchParams = new URLSearchParams({
+      address,
+      parentSubaccountNumber: '0',
+      status: 'OPEN,UNTRIGGERED',
+    })
+    const resourceUrl = `${BASE_INDEXER_URL}/orders/parentSubaccountNumber?${searchParams.toString()}`
+
+    try {
+      const response = await fetch(resourceUrl, {
+        method: 'GET',
+        headers,
+      })
```

Comment:
@hecticme 

Since this is for just study Object-Oriented Programming, does not require to change it now

```suggestion
    const fetchOptionHash = this.generateFetchOptionHash({
      address,
    })

    try {
      const response = await fetch(resourceUrl, fetchOptionHash)
```